python/ray/tests/test_placement_group_mini_integration.py

```diff
@@ -23,7 +23,7 @@ def run_mini_integration_test(cluster, pg_removal=True, num_pgs=999):
     # When 999 resource quantity is used, it fails about every other time
     # when the test was written.
     resource_quantity = num_pgs
-    num_nodes = 5
+    num_nodes = 4
```

Comment:
What's the reason for this change?

Reply:
It seems because unit test VMs are very limited in CPU, lowering CPU priority of workers resulted in some workers unable to register with Raylet. I'm trying to see if reducing the number of workers somewhat can fix the issue. And I will try calling `nice()` after coreworker registers with raylet.

Reply:
Got it, thanks!